CHANGELOG.md

```diff
@@ -21,6 +22,7 @@ Monitor the release status by regions at [AKS-Release-Tracker](https://releases.
 * [OpenID Connect (OIDC) issuer](https://learn.microsoft.com/azure/aks/use-oidc-issuer) is now enabled by default on new cluster creation for Kubernetes version 1.34 and above.
 * [Node Auto-provisioning](https://learn.microsoft.com/azure/aks/node-autoprovision?tabs=azure-cli) enabled clusters can use [planned maintenance](https://learn.microsoft.com/azure/aks/planned-maintenance?tabs=azure-cli) for scheduling node image upgrades that adhere to `aksManagedNodeOSUpgradeSchedule`.
 * When upgrading from kubenet to Azure CNI Overlay, customers can now specify a different pod CIDR using the --pod-cidr parameter. See [Upgrade Azure CNI](https://docs.azure.cn/aks/upgrade-azure-cni#upgrade-to-azure-cni-overlay) for more information.
+* Security Patch in [AKS-Release-Tracker](https://releases.aks.azure.com/) now shows versions for Azure Linux.
```

Comment:
this should be under announcements section as it's not an AKS cluster enhancement and instead a release tracker enhancement.

on "now shows versions for Azure Linux", clarifying question - did we just add 'v3' under security patch? or was Azure Linux not there before at all and we added Azure Linux v3 there under security patch? if it's the former, may be we can skip this line as we only show info on one version of Azure Linux under release tracker as of now. if it's the latter, then we should reword to mention that Security Patch tab under AKS release tracker now provides information for Azure Linux v3.